mscore/widgets/alignSelect.cpp

```diff
@@ -35,13 +35,13 @@ AlignSelect::AlignSelect(QWidget* parent)
     g2->addButton(alignBaseline);
     g2->addButton(alignBottom);
 
-    alignLeft->setIcon(*InspectorIconLoader::icon(InspectorIconLoader::TEXT_JUSTIFY_LEFT_ICON));
-    alignRight->setIcon(*InspectorIconLoader::icon(InspectorIconLoader::TEXT_JUSTIFY_RIGHT_ICON));
-    alignHCenter->setIcon(*InspectorIconLoader::icon(InspectorIconLoader::TEXT_JUSTIFY_CENTER_ICON));
-    alignVCenter->setIcon(*InspectorIconLoader::icon(InspectorIconLoader::TEXT_ALIGN_V_CENTER_ICON));
-    alignTop->setIcon(*InspectorIconLoader::icon(InspectorIconLoader::TEXT_ALIGN_V_TOP_ICON));
-    alignBaseline->setIcon(*InspectorIconLoader::icon(InspectorIconLoader::TEXT_ALIGN_V_BASELINE_ICON));
-    alignBottom->setIcon(*InspectorIconLoader::icon(InspectorIconLoader::TEXT_ALIGN_V_BOTTOM_ICON));
+      alignLeft->setIcon(*icons[static_cast<int>(Icons::textLeft_ICON)]);
+      alignRight->setIcon(*icons[static_cast<int>(Icons::textRight_ICON)]);
+      alignHCenter->setIcon(*icons[static_cast<int>(Icons::textCenter_ICON)]);
+      alignVCenter->setIcon(*icons[static_cast<int>(Icons::textVCenter_ICON)]);
+      alignTop->setIcon(*icons[static_cast<int>(Icons::textTop_ICON)]);
+      alignBaseline->setIcon(*icons[static_cast<int>(Icons::textBaseline_ICON)]);
+      alignBottom->setIcon(*icons[static_cast<int>(Icons::textBottom_ICON)]);
```

Comment:
I think better use `int(Icons::textTop_ICON)` instead of `static_cast<int>(Icons::textTop_ICON)`

Reply:
There is no need to use C-style cast here, it's much safer to use static_cast
At least, we should do this because it's one of the rules from the our code-style, you know)

Reply:
int(x) - not C-style cast, it safe as static_cast
(int)x - this is C-style cast :)

Reply:
nope, it's an equivalent of the C-style cast
https://en.cppreference.com/w/cpp/language/explicit_cast